packages/relay/src/lib/precheck.ts

```diff
@@ -376,4 +380,17 @@ export class Precheck {
       throw predefined.UNSUPPORTED_TRANSACTION_TYPE;
     }
   }
+
+  /**
+   * Checks if the receiver account exists.
+   * @param {Transaction} tx - The transaction.
+   * @param {RequestDetails} requestDetails - The request details for logging and tracking.
+   */
+  async receiverAccount(tx: Transaction, requestDetails: RequestDetails) {
+    const verifyAccount = await this.mirrorNodeClient.getAccount(tx.to!, requestDetails);
```

Comment:
Let's add the parsedTx.to check here. We can make it only run this precheck of parsedTx.to is found so in line 390 we don't have to use the `!` operator.